packages/block-editor/src/components/block-variation-transforms/index.js

```diff
@@ -95,6 +98,49 @@ function VariationsDropdown( {
 	);
 }
 
+function VariationsToggleGroupControl( {
+	className,
+	onSelectVariation,
+	selectedValue,
+	variations,
+} ) {
+	return (
+		<fieldset className={ className }>
```

Comment:
Is this fieldset necessary? The ToggleGroupControl alone is semantically grouped and labeled. Maybe swap with a `div` if it's just for the wrapper classes?